main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
```

Comment:
If you are considering the while loop, it is better to use a more descriptive variable names.